code/modules/antagonists/changeling/powers/fakedeath.dm

```diff
@@ -1,13 +1,46 @@
-/obj/effect/proc_holder/changeling/fakedeath
+/datum/action/changeling/fakedeath
 	name = "Reviving Stasis"
-	desc = "We fall into a stasis, allowing us to regenerate and trick our enemies."
+	desc = "We fall into a stasis, allowing us to regenerate and trick our enemies. Costs 15 chemicals."
+	button_icon_state = "fake_death"
 	chemical_cost = 15
 	dna_cost = 0
 	req_dna = 1
 	req_stat = DEAD
+	ignores_fakedeath = FALSE

```

Comment:
now that I think about it this has other checks meaning this can probably always be true